src/domain/authentication/cookie.ts

```diff
@@ -0,0 +1,11 @@
+export interface ICookieOptions {
```

Comment:
everything optional is not good practice. is that intentional? or just an oversight?

Reply:
yeah all the fields for Cookie Options are optional. You can look at how Express implements CookieOptions here https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express-serve-static-core/index.d.ts#L350-L360.

I did not want to couple the kratos service with express so i just created a common domain model for CookieOptions